### DISCUSSION TITLE
I cant find Dockerfile to build mr

### DISCUSSION THREAD

**Discussion by jordancabral** [featurehub-io/featurehub#420] [Q&A] [answered]
Hello, im working in some minor changes in our featurehub implementation.
I already build de project, but i cant find the Dockerfile to build de mr service.
There is one here https://github.com/featurehub-io/featurehub/tree/master/pipeline/build/base_mr, but it looks like never copy de builded jar file, i want to build an image like the one is in docker hub (featurehub/mr), but with my own local build. 
I can make my own but i need to know how to config the entrypoint.

Thanks

**Comment by rvowles** [comment, marked as answer]
Hi Jordon, 

There is no Dockerfile for building the MR, Party-Server, Dacha or Edge servers, that is done by the Google Jib Maven plugin that lets us build these docker images without Docker. Its defined in the backend/tile-app/tile.xml file if that helps?

**Reply by jordancabral** [reply]
Thanks i will check that file

**Reply by jordancabral** [reply]
Thanks, with this info i am able to build my docker image.
